src/service/tiki-service/internal-handlers.ts

```diff
@@ -54,7 +54,7 @@ export class InternalHandlers {
     if (!receipt.blinkReceiptId) {
       console.warn(`Receipt missing ID. Skipping: ${JSON.stringify(receipt)}`);
     } else {
-      this.store.receipt.add(receipt.blinkReceiptId).catch((error): void => {
+      this.store.receipt.add(receipt.blinkReceiptId, new Date(receipt.receiptDate?.value!)).catch((error): void => {
```

Comment:
I think we want to use `receiptDateTime: number;` to avoid timezone issues.

Reply:
updated!